src/vue/backend/src/router/index.js

```diff
@@ -89,27 +82,27 @@ const routes = [
             {
                 path: '/guided-setup',
                 name: 'Guided Setup',
-                component: GuidedSetup,
+                component: Generic,
                 children: [
                     {
-                        path: 'connect',
-                        children: [
-                            {
-                                path: 'api',
-                                name: 'Connect Account',
-                                component: GuidedSetupConnectApi
-                            },
-                            {
-                                path: 'general',
-                                name: 'General Settings'
-                                // component
-                            },
-                            {
-                                path: 'omnibar',
-                                name: 'Omnibar Settings'
-                                // component
-                            }
-                        ]
+                        path: 'welcome',
+                        name: 'Welcome',
+                        component: () => import('@/views/guided-setup/Welcome')
+                    },
+                    {
+                        path: 'api',
```

Comment:
`api`, `general`, and `omnibar` are missing the `connect/` prefix that visually links them as part of a step